index.html

```diff
@@ -1,184 +1,356 @@
+                  maxlength="30"
+                />
 
-        <section class="about-section">
-            <div class="about-container">
-                <div class="desc">
-                    <h2 class="about-heading">About<br> Myself</h2>
-                    <p class="about-desc">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-                    <a href="#" class="connect">Let's Connect</a>
-                    <div class="social-icon">
-                        <ul>
-                            <li><a href="#"><img src="/img/twiter icon.svg" alt="Twitter icon"></a></li>
-                            <li><a href="#"><img src="/img/Linkedin icon.svg" alt="LinkedIn Icon"></a></li>
-                            <li><a href="#"><img src="/img/medium icon.svg" alt="medium icon"></a></li>
-                            <li><a href="#"><img src="/img/github icon.svg" alt="github icon"></a></li>
-                            <li><a href="#"><img src="/img/hi icon.svg" alt="Hi icon"></a></li>
-                        </ul>
-                    </div>
-                    <button class="btn btn-primary">Get My Resume</button>
-                </div>
-                <div class="skils">
-                    <ul class="skills-items">
-                        <li class="skill-item">
-                            <h3 class="lang">Languages <img src="/img/down arrow.png" alt=""></h3> <br>
-                            <span class="item"> <img src="/img/js.png" alt=""><h4>Javascript</h4></span> <br>                 
-                            <span class="item"> <img src="/img/html.png" alt=""><h4>HTML</h4></span> <br>                 
-                            <span class="item"> <img src="/img/css.png" alt=""><h4>CSS</h4></span> <br>                 
-                        </li>
-                        <li class="skill-item">
-                            <h3 class="lang">Frameworks <img src="/img/Union.png" alt=""></h3> 
-                        </li>
-                        <hr>
-                        <li class="skill-item">
-                            <h3 class="lang">Skills <img src="/img/Union.png" alt=""></h3>
-                        </li>
-                        <hr>
-                    </ul>
-                </div>
-                </div>
-        </section>
+                <textarea
+                  rows="5"
+                  name="message"
+                  id="message"
+                  placeholder="Write your message"
+                  required=""
+                  maxlength="500"
+                ></textarea>
+                <input
+                  type="hidden"
+                  name="_subject"
+                  id="email-subject"
+                  value="Contact Form Submission"
+                  maxlength="30"
+                />
```

Comment:
[OPTIONAL] As suggested by the previous reviewer, please only leave on your code the structure needed for your HTML page. This makes your code a lot easier to understand, maintain and review 😀.